server/web/comment/index.js

```diff
@@ -14,7 +14,7 @@ exports.register = function (server, options, next) {
     },
     path: '/comment/delete/{commentId}',
     handler: function (request, reply) {
-      if (!request.session.get('admin')) {
+      if (!request.yar.get('admin')) {
```

Comment:
[breaking change from yar explained](https://github.com/hapijs/yar/blob/3af58b9c54e13227145238ec83e21e019a33beb8/README.md#upgrading-to-6xx-and-greater)